src/commands/deploy.ts

```diff
@@ -235,32 +578,46 @@ const deployService = async ({
  * @param param0 Readonly<{ deployServiceOptions: DeployServiceOptions; count: number; commandObj: CommandObj;}>
  * @returns Promise<DeployedServiceConfig[] | null>
  */
+/*
 const deployServices = async ({
```

Comment:
Commented out code: either remove or add a note explaining why does it have to be left this way (and when it will be fixed)

Reply:
Also forgot to remove it. Thanks!
I want to write a better deploy by blueprint id in the future by first calculating it locally and then checking if there is already such a blueprint id on the peer and only then deploying